internal/buildscripts/packaging/fpm/etc/otel/collector/fluentd/conf.d/journald.conf

```diff
@@ -25,30 +25,40 @@
 #
 #      systemctl daemon-reload
 #
-# 3. Make any necessary changes to the configuration below and copy/rename this
-#    file to "/etc/otel/collector/fluentd/conf.d/journald.conf".  See
+# 3. Make any necessary changes to the configuration below.  See
 #    https://github.com/fluent-plugin-systemd/fluent-plugin-systemd for
 #    details and other options.
 #
 # 4. Since the fluentd service runs as the "td-agent" user, ensure that this
 #    user has permissions to access the path configured below.  For example, if
-#    the "/run/log/journal" path is only readable by the "systemd-journal"
+#    the "/var/log/journal" path is only readable by the "systemd-journal"
 #    group, it may be necessary to add the "td-agent" user to the group by
 #    running the command:
 #
 #      usermod -a -G systemd-journal td-agent
 #
+#    Alternatively, see https://docs.fluentd.org/deployment/linux-capability
+#    for installing/enabling the Linux capability module on fluentd.
+#
 # 5. Restart the fluentd service to apply the changes by running:
 #
 #      systemctl restart td-agent
 #
 
 <source>
+  @id journald
   @type systemd
   @label @SPLUNK
   tag "journald"
-  path "/run/log/journal"
+  path "/var/log/journal"
+  read_from_head true
```

Comment:
Is this required?

Reply:
Not sure. I just copied that option from https://github.com/signalfx/splunk-otel-collector-chart/blob/main/helm-charts/splunk-otel-collector/templates/configmap-fluentd.yaml#L148.

Reply:
My understanding is that this is for file rotation and startup -- results in the entire thing being read and sent on every startup. I am under the impression we do not want this and that it could lead to performance issues.